src/components/calculator/ActivityRiskControls.tsx

```diff
@@ -131,3 +135,15 @@ export const ActivityRiskControls: React.FunctionComponent<{
     </React.Fragment>
   )
 }
+
+const maskPopover = (
+  <Popover id="popover-basic">
+    <Popover.Title as="h3">Masks</Popover.Title>
+    <Popover.Content>
+      For more details on masks, see&nbsp;
```

Comment:
`see{' '}` is more typically how we would force this space, since it doesn't need to be non-breaking (just needs to exist)

Reply:
Done